trie/trie_reader.go

```diff
@@ -45,14 +46,10 @@ type trieReader struct {
 
 // newTrieReader initializes the trie reader with the given node reader.
 func newTrieReader(stateRoot, owner common.Hash, db *Database) (*trieReader, error) {
-	// if stateRoot == (common.Hash{}) || stateRoot == types.EmptyRootHash {
-	// 	if stateRoot == (common.Hash{}) {
-	// 		log.Error("Zero state root hash!")
-	// 	}
-	// 	return &trieReader{owner: owner}, nil
-	// }
-	if stateRoot == types.EmptyZkTrieRootHash {
-		// log.Error("Zero state root hash!")
+	if stateRoot == (common.Hash{}) || stateRoot == types.EmptyRootHash {
+		if stateRoot == (common.Hash{}) {
+			log.Error("Zero state root hash!")
```

Comment:
isn't zktrie empty root hash `common.Hash{}`? why do we need to log error here?

Reply:
We don't, we can remove it if it ends up being too annoying.